### ISSUE TITLE
Add a test for text that is colored, and suggest that Bold or Italics be applied to it.  With a UFIXIT.  It should be mutually exclusive from the "color contrast" error.

### ISSUE THREAD

**Issue by bagofarms** [ucfcdl/UDOIT#14]

**Comment by techrangerfj** [comment]
I'm assuming this should add a suggestion to the report. I will implement within code base under the name 'cssTextStyleEmphasize'. Would you be able to provide me with the verbiage for the report? @bagofarms

**Comment by bagofarms** [comment]
Avoid using color alone for emphasis.

When emphasizing text, you may use color with sufficient contrast as long as you also apply some other form of emphasis, such as bold or italics.  This ensures that screen reader users are aware of the text's importance.
